src/components/TapeSpinner/index.tsx

```diff
@@ -0,0 +1,31 @@
+export default function TapeSpinner() {
+    return (
+      <>
+      <div className={`w-20 h-[1.5px] border-1px bg-[#000000] mr-4`}>
+
+</div>
```

Comment:
NIT: this empty div (i'm guessing represents the lines) could just be expressed as `<div />` since it has no child elements. Just makes the code ever so slightly cleaner, no functional difference though.